.gitignore

```diff
@@ -4,3 +4,5 @@
 /dist
 *.swp
 /.trash-cache
+/.idea
```

Comment:
Dont bput .idea in the project's .gitignore. put it in your global git ignore file that can live at the root of your home directory.